2025-01-09 15:05:54,899 INFO    MainThread:14361 [wandb_init.py:_log_setup():523] Logging user logs to /Users/guyshani/Documents/PHD/Aim_2/cycle_GAN/wandb/run-20250109_150554-m7w27hzr/logs/debug.log
2025-01-09 15:05:54,899 INFO    MainThread:14361 [wandb_init.py:_log_setup():524] Logging internal logs to /Users/guyshani/Documents/PHD/Aim_2/cycle_GAN/wandb/run-20250109_150554-m7w27hzr/logs/debug-internal.log
2025-01-09 15:05:54,899 INFO    MainThread:14361 [wandb_init.py:init():641] calling init triggers
2025-01-09 15:05:54,899 INFO    MainThread:14361 [wandb_init.py:init():647] wandb.init called with sweep_config: {}
config: {'epochs': 200, 'latent_dim': 64, 'batch_size': 32, 'nb_layers': 2, 'hdim': 256, 'lr': 0.0005, 'nb_critic': 5}
2025-01-09 15:05:54,899 INFO    MainThread:14361 [wandb_init.py:init():666] found existing run on stack: rcmbqqgj
2025-01-09 15:05:54,899 INFO    MainThread:14361 [wandb_run.py:_finish():2140] finishing run guyshani-tel-aviv-university/adversarial_gene_expr/rcmbqqgj
2025-01-09 15:05:54,899 ERROR   MainThread:14361 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-01-09 15:05:54,899 INFO    MainThread:14361 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-01-09 15:05:54,900 INFO    MainThread:14361 [wandb_init.py:_jupyter_teardown():451] cleaning up jupyter logic
2025-01-09 15:05:54,900 INFO    MainThread:14361 [wandb_run.py:_atexit_cleanup():2380] got exitcode: 0
2025-01-09 15:05:54,900 INFO    MainThread:14361 [wandb_run.py:_restore():2362] restore
2025-01-09 15:05:54,900 INFO    MainThread:14361 [wandb_run.py:_restore():2368] restore done
2025-01-09 15:05:59,186 INFO    MainThread:14361 [wandb_run.py:_footer_history_summary_info():3984] rendering history
2025-01-09 15:05:59,187 INFO    MainThread:14361 [wandb_run.py:_footer_history_summary_info():4016] rendering summary
2025-01-09 15:05:59,197 INFO    MainThread:14361 [wandb_run.py:_footer_sync_info():3945] logging synced files
2025-01-09 15:05:59,200 INFO    MainThread:14361 [wandb_init.py:init():674] starting backend
2025-01-09 15:05:59,201 INFO    MainThread:14361 [wandb_init.py:init():678] sending inform_init request
2025-01-09 15:05:59,202 INFO    MainThread:14361 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-01-09 15:05:59,203 INFO    MainThread:14361 [wandb_init.py:init():693] backend started and connected
2025-01-09 15:05:59,204 INFO    MainThread:14361 [wandb_run.py:_label_probe_notebook():1226] probe notebook
2025-01-09 15:05:59,209 INFO    MainThread:14361 [wandb_init.py:init():786] updated telemetry
2025-01-09 15:05:59,262 INFO    MainThread:14361 [wandb_init.py:init():818] communicating run to backend with 90.0 second timeout
2025-01-09 15:05:59,733 INFO    MainThread:14361 [wandb_init.py:init():868] starting run threads in backend
2025-01-09 15:05:59,822 INFO    MainThread:14361 [wandb_run.py:_console_start():2415] atexit reg
2025-01-09 15:05:59,823 INFO    MainThread:14361 [wandb_run.py:_redirect():2265] redirect: wrap_raw
2025-01-09 15:05:59,823 INFO    MainThread:14361 [wandb_run.py:_redirect():2330] Wrapping output streams.
2025-01-09 15:05:59,823 INFO    MainThread:14361 [wandb_run.py:_redirect():2355] Redirects installed.
2025-01-09 15:05:59,824 INFO    MainThread:14361 [wandb_init.py:init():910] run started, returning control to user process
2025-01-09 15:05:59,824 INFO    MainThread:14361 [wandb_run.py:_config_callback():1283] config_cb None None {'selected_categories': ['dataset']}
